### ISSUE TITLE
An input-group with dropdown messes up widths inside btn-toolbar

### ISSUE THREAD

**Issue by cervengoc** [twbs/bootstrap#20366]
I have a `btn-toolbar`. I have an `input-group` with an addon and a dropdown menu inside it, but it displays incorrectly.

Here is a sapmle to reproduce: http://www.bootply.com/jGeX54wf95

If I turn ott the `width: 1%` rule for the `input-group-addon`, it displays correctly.

**Comment by twbs-lmvtfy** [comment]
Hi @cervengoc!

You appear to have posted a live example (http://s.bootply.com/render/jGeX54wf95), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 5, column 9 thru column 30: A document must not include both a ```` meta ```` element with an ```` http-equiv ```` attribute whose value is ```` content-type ````, and a ```` meta ```` element with a ```` charset ```` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by wolfy1339** [comment]
The error seems to be from Bootply, just to conirm that that isn't the problem (which it shouldn't), can you please re-test using this JSBin: http://output.jsbin.com/woqizukufu

**Comment by cervengoc** [comment]
It's the same, width of `input-group-addon` is huge, and the `input-group` does not fit into the `toolbar` but have it's own row.
By the way, I use version 3.3.7 on Windows 10 and Chrome 51 (latest).

**Comment by cervengoc** [comment]
I see that this scenario is kind of violating the `input-group` structure as it does not have any `input`, but a textual addon to a dropdown is quite common use case in my opinion. I'm open to any workaround though.

**Comment by cvrebert** [comment]
Fairly sure that putting an `.input-group` in a `.btn-toolbar` isn't permissible.

**Comment by cervengoc** [comment]
@cvrebert It might be true but I've seen styling for that in `button-groups.less`, that's why I thought it should work.

**Comment by cervengoc** [comment]
I've found that adding `width: 1%` to the `input-group` solves the problem, at least in Chrome 51. Not sure however if it's a good thing, I was just experimenting.

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
